package.json

```diff
@@ -35,7 +35,12 @@
     "clean:example": "rm -rf build.log example/assets/*",
     "coverage": "nyc report --reporter=text-lcov > coverage.lcov",
     "docs": "npm run build:docs && open docs/index.html",
+    "format": "prettier --write '{src,test}/**/*.js'",
+    "format:check": "prettier --debug-check --list-different '{src,test}/**/*.js'",
+    "format:lint": "yarn format && yarn lint:fix",
+    "format:check:lint": "yarn format:check && yarn lint",
```

Comment:
@jrajav and here is the script that will fail if there is a formatting or linting error.  this will be useful as a failure condition for CI builds.